rental_module.rb

```diff
@@ -0,0 +1,28 @@
+require './rental'
+require './classroom'
+
+module NewRental
+  def create_rentals
+    print "Select a book from the folowing list by number\n"
+    @books.each_with_index { |book, index| puts "#{index}) Title: #{book.title}, Author: #{book.author}" }
+    book_index = gets.chomp.to_i
+    print "Select a person from the folowing list by number (not id)\n"
+    @people.each_with_index { |p, index| puts "#{index}) [#{p.class}] Name: #{p.name}, Age: #{p.age}, ID: #{p.id}" }
+    person_index = gets.chomp.to_i
+    print 'Date: '
+    date = gets.chomp
+    rental = Rental.new(date, @books[book_index], @people[person_index])
+    @rentals << rental
+    puts 'Rental created successfully'
+  end
+
+  def list_rentals
+    puts 'ID of person:'
+    id = gets.chomp
+    puts 'Rentals:'
+    rentals = @rentals.select { |rental| rental.person.id == id.to_i }
+    rentals.each { |rent| puts "Date: #{rent.date}, Book #{rent.book.title} by #{rent.book.author}" }
+  end
```

Comment:
- [ ] good job on cleaning up the `main.rb` file 👍 however this method is not according to the "single responsibility principle" all requests for input and input validation should've been placed in classes/methods whose responsibilities are to request or validate input. Please refer to my previous comment and [this article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid) for example and explanation.

Reply:
Thank you for your valuable comment, we solved it as you can see in the image below![image](https://user-images.githubusercontent.com/60292761/141952277-a2e6363a-bede-460d-9135-f610de93bd71.png)